client/post-editor/editor-sidebar/index.jsx

```diff
@@ -10,7 +10,26 @@ import EditorDrawer from 'post-editor/editor-drawer';
 import EditorSidebarHeader from './header';
 import SidebarFooter from 'layout/sidebar/footer';
 import EditorActionBar from 'post-editor/editor-action-bar';
+import EditorFeedbackInvitation from 'post-editor/editor-feedback-invitation';
 import EditorDeletePost from 'post-editor/editor-delete-post';
+import FeedbackView from './feedback-view';
+
+const mockSharedLinks = [
+	{
+		label: 'email1@share.test',
+		link: 'http://share-link-1',
+		comments: [
+			'Comment one',
+			'Comment two',
+			'Three-score and two comments ago... there was nothing. The post was formless and void.',
+		],
+	},
+	{
+		label: 'email2@share.test',
```

Comment:
fun fact! any `@example.com` email address is by-spec not routable, meaning you can safely use them without worrying that someone might by happenstance actually have that email address.